pyseir/deployment/demographic_mapper.py

```diff
@@ -0,0 +1,729 @@
+        Rates of hospitalization by age group, type of hospitalization,
+        and unit of rates, with type of hospitalization as primary key,
+        unit as secondary key, and array of corresponding time series of
+        rates as values.
+        For example, IHR['HGen']['per_capita'] is the
+        time series of probability of being admitted to non-ICU per
+        capita among infected population (asymptomatic + symptomatic).
+    IFR: dict
+        Rates of mortality by age group, type of hospitalization,
+        and unit of rates, with type of hospitalization as primary key,
+        unit as secondary key, and array of corresponding time series of
+        rates as values.
+        For example, IFR['HICU']['per_capita'] is the time
+        series of probability of death in ICU per capita among infected
+        population (asymptomatic + symptomatic + hospitalized).
+    prevalence: np.array
+        Age-specific prevalence time series simulated with SEIR model
+        with MLE parameters.
+    results: dict
+        Contains:
+            - compartments:
+              - <compartment>: time series of population at a specific
+                               infection state (susceptible, infected,
+                               hospitalized, etc.) simulated by MLE model
+                               assuming the population has the demographic
+                               distribution of the target population. Each time
+                               series is recorded as pd.DataFrame, with dates of
+                               prediction as index.
+              name of compartment include: S - susceptible, E - exposed,
+              A - asymptomatic, I - symptomatic, HGen - in non-ICU, HICU -
+              in ICU, HVent - on ventilator, N - entire population.
+            - <measure>:
+              - <measure_unit>: time series of covid measures predicted using
+                                the MLE model and averaged over target
+                                demographic distribution (adjusted by risk
+                                modification if relative risk is specified).
+                                The time series is recorded as pd.DataFrame,
+                                with dates of prediction as index.
+
+
+
+    Parameters
+    ----------
+    fips: str
+        State of county FIPS code
+    mle_model: SEIRModelAge
+        Model with age structure and MLE model parameters
+    fit_results: dict
+        MLE epi parameters and associated errors. The parameter used is
+        t0_date.
+    measures: str or list(str)
+        Names of covid measures, should be interpretable by CovidMeasure.
+    measure_units: str or list(str)
+        Units of covid measures, should be interpretable by CovidMeasureUnit.
+    target_age_distribution_pdf: callable
+        Function that takes an array of age as input returns PDF of the age
+        based on the age distribution of target population.
```

Comment:
This description could be fleshed out a little more. For example, what age ranges need to be handled, and at what granularity? Are there certain age bins that are expected?